apis/v1alpha1/httproute_types.go

```diff
@@ -500,6 +500,15 @@ type HTTPRouteForwardTo struct {
 	// The gateway status for this route should be updated with a
 	// condition that describes the error more specifically.
 	//
+	// The protocol to use is defined using AppProtocol field (introduced in
+	// Kubernetes 1.18) in the Service resource. In the absence of the
+	// AppProtocol field a `networking.x-k8s.io/app-protocol` annotation on the
```

Comment:
Sorry, one more nit. I think all that's really needed is a `the` in front of `AppProtocol`, but I also experimented with a slightly different way to phrase the first two sentences, not 100% sure it's better.

```suggestion
	// The protocol to use should be specified with the AppProtocol field on Service
	// resources. This field was introduced in Kubernetes 1.18. If using an earlier version
	// of Kubernetes, a `networking.x-k8s.io/app-protocol` annotation on the
```